DotNut/ProofSelector.cs

```diff
@@ -0,0 +1,404 @@
+using System.Diagnostics;
+
+namespace DotNut;
+
+public class SendResponse
+{
+    public List<Proof> Keep { get; set; } = new();
+    public List<Proof> Send { get; set; } = new();
+}
+
+// Borrowed from cashu-ts 
+// see https://github.com/cashubtc/cashu-ts/pull/314
+public class ProofSelector
```

Comment:
Could be good to have an interface so that wallets can swap selecting algos easily